spec/unit/gdor_mods_fields_spec.rb

```diff
@@ -8,7 +8,9 @@
   end
 
   let :logger do
-    Logger.new StringIO.new
+    lgr = Logger.new(StringIO.new)
```

Comment:
An idiomatic way to write this might be:

```
Logger.new(StringIO.new).tap { |lgr| lgr.level = Logger::WARN }
```

That said, does this test (and the similar one below..) even check the output of the logger?